Documentation/manpages/dotnet-native.1

```diff
@@ -0,0 +1,57 @@
+.\" Automatically generated by Pandoc 1.15.1
```

Comment:
This one is tricky, I will leave as-is, but it should become a part of the dotnet-compile man page.